words.html

```diff
@@ -403,447 +407,445 @@
+a co ze <i>zmianą w samej grze?</i>
+Co mogłoby prowadzić do...
 </p>
 <p id="evo_11_btn">
-...the evolution of <i>distrust?</i>
+...ewolucji <i>nieufności?</i>
 </p>
 
 <!-- - - - - - - - - - - - - - - - - -->
 <!-- - - - - - DISTRUST - - - - - - -->
 <!-- - - - - - - - - - - - - - - - - -->
 
 <p id="distrust_1">
-Before everything goes to heck, let's start with something nice!
-Here's a world filled entirely with <span class="all_c">Always Cooperates</span>,
-except for one <span class="all_d">Always Cheat</span> and one <span class="tft">Copycat</span>.
+Zanim wszystko się zawali, zacznijmy od czegoś miłego!
+Oto świat wypełniony całkowicie <span class="all_c">Ufnymi Wspólnikami</span>,
+z wyjątkiem jednego <span class="all_d">Notorycznego Oszusta</span> i jednej <span class="tft">Papugi</span>.
 <br><br>
-Use the buttons on the right to <b>START</b> the sim,
-go through it <b>step-by-step</b>, or <b>reset</b> it. &rarr;
+Użyj przycisków po prawej aby <b>ROZPOCZĄĆ</b> symulację,
+przechodzić przez nią <b>krok-po-kroku</b>, lub ją <b>zresetować</b>. &rarr;
 </p>
 
 <p id="distrust_2">
-As you already know, <span class="tft">Copycat</span> wins handily in the long run,
-under our current rules!
+Jak już wiesz, <span class="tft">Papugi</span> na dłuższą metę z łatwością wygrywają...
 <br><br>
-But that's under our <i>current</i> rules, which say that
-players play against each other for <i>10</i> rounds per match.
-Does <span class="tft">Copycat</span> still win at 7 rounds? 5 rounds? 3? 2? 1?
+
+...zgodnie z <i>aktualnymi</i> zasadami,
+które mówią, że gracze toczą przeciwko sobie 10 rund w każdym meczu.
+Czy <span class="tft">Papugi</span> wciąż wygrywają po 7 rundach? 5 rundach? 3? 2? 1?
 <br><br>
-<b>Change the number of rounds</b> with the slider below,
-then <b>START</b> the sim, and see what happens.
-Feel free to experiment!
+<b>Zmień liczbę rund</b>, używając suwaka poniżej,
+następnie <b>ROZPOCZNIJ</b> symulację i zobacz co się stanie.
+Poeksperymentuj trochę!
 </p>
 
 <p id="distrust_2_end">
-once you're done playing around, click:
+gdy już skończysz zabawę, kliknij:
 </p>
 
 <p id="distrust_3">
-As you saw, if you don't play enough rounds, (here: 5 or less)
-<span class="all_d">ALWAYS CHEAT</span> dominates.
+Jak pewnie zauważyłeś, jeśli nie zagrasz wystarczającej ilości rund (przynajmniej 5),
+<span class="all_d">Notoryczny Oszust</span> będzie dominować.
 <br><br>
-In 1985, when Americans were asked how many close friends they had,
-the most common answer was "three". In 2004, the most common answer was <i>"zero"</i>.
-We now have fewer friends across class, racial, economic, and political lines,
-because we have fewer friends -- <i>period.</i>
-And as you just discovered for yourself,
-<b>the fewer "repeat interactions" there are, the more distrust will spread.</b>
+W 1985 roku, gdy Amerykanie zostali zapytani jak wielu mają przyjaciół,
+najczęstszą odpowiedzią było "trzech". W 2004, najczęstszą odpowiedzią było <i>"zero"</i>.
```

Comment:
Ogólnie spoko 👍  , ale to zero jakoś bardziej do mnie przemawia, jest bardziej przerażające niż "żadnych" :(